src/Mastodon/ApiUrl.elm

```diff
@@ -69,6 +70,22 @@ userAccount server =
     server ++ accounts ++ "verify_credentials"
 
 
+searchAccount : Server -> String -> Int -> Bool -> String
+searchAccount server query limit resolve =
+    server
+        ++ accounts
+        ++ "search?q="
+        ++ query
+        ++ "&limit="
+        ++ (toString limit)
+        ++ "&resolve="
+        ++ (if resolve then
+                "true"
+            else
+                "false"
+           )
```

Comment:
Look at `encodeUrl` in the `Mastodon.Encode` module, it may be a little nicer here.